### PR TITLE
Calculator Assignment Code Review

### PR DESCRIPTION
**Pull request by aruh21**
@RatiKulkarniWH Please review my Calculator assignment and do let me know If I need make changes.

### CODE REVIEW

File: ArithMaticOperation.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+public class ArithMaticOperation {
```
Comment (RatiKulkarniWH) [inline ArithMaticOperation.txt:1]: @aruh21 These are not actual code files. I can see all files has been uploaded as text file. Please correct the same and send new review request.

File: ArithMaticOperation.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        }else if(choice == 'sub'){
+            result = num1 - num2 ;
+        }else if(choice == 'mul'){
+            result = num1 * num2 ;
+        } else if(choice == 'div'){
+            result = num1 / num2 ;
+        } else {
+            result = 0;
+        }    
+
```
Comment (RatiKulkarniWH) [inline ArithMaticOperation.txt:19]: These are all un wanted empty space, please remove the same and intend the code properly

- [ ] Remove empty lines
- [ ] Apply proper indentation

**Review by RatiKulkarniWH — CHANGES_REQUESTED** [review]
@aruh21 please work on suggested changes and revert as soon as it is completed.

File: ArithMaticOperation.txt (hunk 1 above)
Comment (aruh21) [reply ArithMaticOperation.txt:1]: sure I will update this branch with proper class and vf page files

File: ArithMaticOperation.txt (hunk 2 above)
Comment (aruh21) [reply ArithMaticOperation.txt:19]: @RatiKulkarniWH I have updated the code.

**Pull request closed without merging**